### ISSUE TITLE
Add a CHANGELOG.md

### ISSUE THREAD

**Issue by bcomnes** [shama/bel#89]
Hey @shama would you be open to me adding a changelog.md to the project?

**Comment by Haroenv** [comment]
Otherwise just telling here what the breaking changes from v4 -> v5 are would be useful

**Comment by shama** [comment]
@bcomnes Yep! If you think it would be helpful. Feel free to do whatever you think is best for this project. Thanks a bunch for all your help!
